### PR TITLE
[Readme] Updated

### PR DESCRIPTION
**Pull request by robertbutacu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,4 +46,275 @@ $ sbt it:test
...
+
+ Transit state:
+   - one of the values active, backlog
+   
+## Endpoints
+
+
+## Production endpoints
+
+```POST    /transits```
```
Comment (udinobi) [inline README.md]: What do you think to prefix all endpoints with `/common-transit-convention` ?

File: README.md [hunk 2]
```diff
@@ -46,4 +46,275 @@ $ sbt it:test
...
+
+## Testing endpoints
+
+```DELETE /test-only/clear```
+
+Clear both repositories - the transit and cached ones.
+
+```GET /test-only/populate```
+
+With query parameters `enRoute` and `arrived`, it populates the database with `x` number of `enRoute` records and `y` number of arrived records.
```
Comment (udinobi) [inline README.md]: Maybe you could also specify that not using the optional params, the `populate` by default create 100 **enRoute** transits and 100 **arrived** transits.

**Comment by udinobi** [comment]
Nice Job!

File: README.md (hunk 2 above)
Comment (robertbutacu) [reply README.md]: Hmm, looking over the code, isnt the default value 0? 100 is default on the frontend
I can set the default to 100 on the BE as well

File: README.md (hunk 1 above)
Comment (robertbutacu) [reply README.md]: Yess, thats a good idea

File: README.md (hunk 2 above)
Comment (udinobi) [reply README.md]: At the end, we need to notify the behavior of the API to the user, and the user mainly will use it via browser (FE) and not via BE. The default for the BE is accordingly not really relevant. You can still set it to 100 if you think that's more consistent.

Comment (robertbutacu) [reply README.md]: Ah, okay - for now, i set it to 100. Also, the default values are specified in the readme

File: README.md [hunk 3]
```diff
@@ -46,4 +46,276 @@ $ sbt it:test
...
+## Testing endpoints
+
+```DELETE /common-transit-convention/test-only/clear```
+
+Clear both repositories - the transit and cached ones.
+
+```GET /common-transit-convention/test-only/populate```
+
+With query parameters `enRoute` and `arrived`, it populates the database with `x` number of `enRoute` records and `y` number of arrived records.
+The default values are
```
Comment (udinobi) [inline README.md]: The default values are... ?

Comment (robertbutacu) [reply README.md]: haha, updated now, mb

**Pull request merged**